### PR TITLE
Spanish self serve

### PR DESCRIPTION
**Pull request by AlfonsoSB**

### CODE REVIEW

**Comment by greysteil** [comment]
@harrison - did you or someone from team front-end have a look over this before it was merged?

@AlfonsoSB - we always get someone else from the team to look over changes to our code and give them a :+1: before merging.

**Comment by AlfonsoSB** [comment]
Didn't realize I had merged it. Won't happen again. 




Thanks for letting me know



—
Enviado desde mi iPhone

File: app/messages/es-ES.js [hunk 1]
```diff
@@ -20,4 +20,10 @@ export default {
     per_transaction_amount_normal: '1%',
     normal_capped_at_amount: '200€',
   },
+  signin: {
+    path: 'https://manage.gocardless.com',
+  },
+  signup: {
+    path: 'https://manage.gocardless.com/signup',
```
Comment (Ken-Lo) [inline app/messages/es-ES.js]: Needs to be /signup?lang=es following https://github.com/gocardless/splash-pages/pull/305

File: app/pages/features/features.es.js [hunk 2]
```diff
@@ -0,0 +1,401 @@
...
+                        Solo necesitas una cuenta bancaria en <Message pointer='country' />. <Link to='contact_sales'>
+                        Contáctanos</Link> para comenzar.
+                      </p>
+                    </IfLocale>
+                    <IfLocale hasInstantSignup>
+                      <div className='u-text-heading u-color-heading u-text-light u-text-no-smoothing u-text-s u-margin-Tm'>
+                        Comienza en minutos
+                      </div>
+                      <p className='u-color-p u-margin-Txs'>
+                          Sólo necesitas una cuenta bancaria en <Message pointer='country' />.
```
Comment (Ken-Lo) [inline app/pages/features/features.es.js]: Indentation

File: app/pages/features/features.es.js [hunk 3]
```diff
@@ -0,0 +1,401 @@
...
+                    <figure className='svg-icon svg-icon--small'>
+                      <CustomerIcon className='svg-icon__image svg-icon__image--small svg-icon__image--small-shadow u-fill-green' />
+                      <CustomerIcon className='svg-icon__image svg-icon__image--small u-fill-dark-gray' />
+                    </figure>
+                    <div className='u-text-heading u-color-heading u-text-light u-text-no-smoothing u-text-s u-margin-Tm'>
+                      Alta retención de usuarios
+                    </div>
+                    <p className='u-color-p u-margin-Txs'>
+                      Como está construido sobre el sistema de Adeudos Directos, con Gocardless
+                       no perderás clientes debido a la caducidad de las tajertas o cancelaciones.
```
Comment (Ken-Lo) [inline app/pages/features/features.es.js]: Indentation

File: app/pages/features/features.es.js [hunk 4]
```diff
@@ -0,0 +1,401 @@
...
+                    <figure className='svg-icon svg-icon--small'>
+                      <DeveloperIcon className='svg-icon__image svg-icon__image--small svg-icon__image--small-shadow u-fill-yellow' />
+                      <DeveloperIcon className='svg-icon__image svg-icon__image--small u-fill-dark-gray' />
+                    </figure>
+                    <div className='u-text-heading u-color-heading u-text-light u-text-no-smoothing u-text-s u-margin-Tm'>
+                      Una API RESTful moderna
+                    </div>
+                      <p className='u-color-p u-margin-Txs'>
+                        Añade GoCardless a tu web o app en minutos con nuestras librerias API.
+                        <a href='https://developer.gocardless.com/pro'> Descubre más</a>.
```
Comment (Ken-Lo) [inline app/pages/features/features.es.js]: Extra space at start of hyperlink

File: app/pages/features/features.es.js [hunk 5]
```diff
@@ -0,0 +1,401 @@
...
+                  <img src='/images/products/real-time-reports.jpg' className='basic-page__real-time-reports-image' />
+                </div>
+                <div className='grid__cell u-size-1of2 u-margin-Tl'>
+                  <h2 className='u-text-heading u-color-heading u-text-l u-text-light u-margin-Bl u-margin-Ts'>
+                    Asegura tu flujo de caja
+                  </h2>
+                  <h3 className='u-text-s'>Sigue el estado de tus pagos en todo momento</h3>
+                  <p className='u-size-5of6 u-color-p u-margin-Bm'>
+                    Nuestro panel de control y notificaciones por email te mantendran informado totalmente
+                     acerca de todos tus cobros y clientes.
```
Comment (Ken-Lo) [inline app/pages/features/features.es.js]: Indentation

File: app/pages/features/features.es.js [hunk 6]
```diff
@@ -0,0 +1,401 @@
...
+            <hr className='u-margin-An' />
+            <div className='site-container u-text-center u-padding-Vxxl'>
+              <div className='grid u-padding-Vxl'>
+                <div className='grid__cell u-size-1of2'>
+                  <h2 className='u-text-l u-text-heading u-color-heading u-text-light u-margin-Bl'>
+                    A tus clientes les encantará
+                  </h2>
+                    <p className='u-size-5of6 u-color-p u-margin-Bm'>
+                      GoCardless es una institución autorizada por la FCA que facilita y
+                       agiliza los cobros a ti y a tus clientes.
```
Comment (Ken-Lo) [inline app/pages/features/features.es.js]: Indentation

File: app/pages/features/features.es.js [hunk 7]
```diff
@@ -0,0 +1,401 @@
...
+                    </p>
+                  <h3 className='u-text-s'>Con un sencillo Setup Online</h3>
+                  <p className='u-size-5of6 u-color-p u-margin-Bm'>
+                    Todo el proceso es online. Utilizando nuestro mandato online, tus clientes podrán
+                     autorizar cobros en minutos - incluso desde su teléfono móvil.
```
Comment (Ken-Lo) [inline app/pages/features/features.es.js]: Indentation

File: app/pages/features/features.es.js [hunk 8]
```diff
@@ -0,0 +1,401 @@
...
+                    <figure className='svg-icon u-center'>
+                      <TickIcon className='svg-icon__image svg-icon__image--shadow u-fill-blue' />
+                      <TickIcon className='svg-icon__image u-fill-dark-gray' />
+                    </figure>
+                    <div className='u-text-heading u-color-heading u-text-light u-text-no-smoothing u-text-m u-margin-Tm'>
+                      Nunca estarás solo
+                    </div>
+                    <p className='u-size-8of10 u-center u-color-p u-margin-Txs'>
+                      Hemos ayudado a cientos de empresas y podemos guiarte en los pasos necesarios para <br />conseguir
+                       que tus clientes cambien.
```
Comment (Ken-Lo) [inline app/pages/features/features.es.js]: Indentation

File: app/pages/features/features.es.js [hunk 9]
```diff
@@ -0,0 +1,401 @@
...
+            <hr className='u-margin-An' />
+            <div className='site-container u-text-center u-padding-Vxxl'>
+              <div className='u-padding-Vxl'>
+                <div className='u-size-1of3 u-center'>
+                  <figure className='svg-icon u-center'>
+                    <PhoneIcon className='svg-icon__image svg-icon__image--shadow u-fill-blue' />
+                    <PhoneIcon className='svg-icon__image u-fill-dark-gray' />
+                  </figure>
+                  <h2 className='u-text-heading u-color-heading u-text-l u-text-light u-margin-Tm'>
+                    <Message pointer="phone_local" />
```
Comment (Ken-Lo) [inline app/pages/features/features.es.js]: Single quotes

File: app/pages/features/features.es.js (hunk 4 above)
Comment (AlfonsoSB) [inline app/pages/features/features.es.js]: That is ok. It doesn't show allright if not there.

File: app/messages/es-ES.js (hunk 1 above)
Comment (AlfonsoSB) [inline app/messages/es-ES.js]: is this online now? I cannot replicate it

File: app/pages/pro/pro.es.js [hunk 10]
```diff
@@ -172,23 +173,26 @@ export default class ProEs extends React.Component {
...
-                      Domiciliaciones Bancarias desde el Reino Unido y la Eurozona en una sola integración
-                      Con nuestra API, tu empresa podra realizar cobros de más de 500 milllones de personas
-                      en 22 países de Europa.
-                    </p>
-                  </div>
+            <div id='forOptimizely-Table'>
+              <div className='u-text-center u-padding-Vxxl u-background-light-gray'>
+                <div className='site-container u-padding-Vxl'>
+                  <ProductComparison />
+                  <table className='u-size-full'><tfoot>
```
Comment (Ken-Lo) [inline app/pages/pro/pro.es.js]: Not sure why there is a `<tfoot>` here?

**Comment by jamesshedden** [comment]
@greysteil @AlfonsoSB @Ken-Lo 

FYI, just looked over this and all looks good minus @Ken-Lo's feedback which I followed up in a separate PR at https://github.com/gocardless/splash-pages/pull/314. As that one's been merged, all this work has also now been deployed :+1:

**Comment by jamesshedden** [comment]
PS @AlfonsoSB Awesome work! :zap: :zap: :zap:

**Comment by AlfonsoSB** [comment]
Thanks!!!!!!!!!! Couldn't do it without you

**Pull request merged**